### ISSUE TITLE
[2017.7.8] test_vultr failing

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt-jenkins#1049]
The following test: `integration.cloud.providers.test_vultr.VultrTest.test_instance` is failing on the cloud tests for py2 and py3

Heres the failure:

```
Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/cloud/providers/test_vultr.py", line 163, in test_instance
    [i.strip() for i in create_vm]
AssertionError: 'CLOUD-TEST-RUYXBX' not found in []
```

job: https://jenkinsci.saltstack.com/job/2017.7.8/view/Cloud/job/salt-centos-7-py2-cloud/2/testReport/junit/integration.cloud.providers.test_vultr/VultrTest/test_instance/

**Comment by Ch3LL** [comment]
okay i think this is resolved. There was some cloud-test VMS that were never cleaned up. I removed those and now we can build. I will need to monitor to make sure the future VMs are getting deleted. Wont close until i see them pass during a test run

**Comment by Ch3LL** [comment]
ahh i saw an error in the UI. When i tried to delete the VM right after creation it said something about it can't remove the VM until 5 minutes after creation. Maybe we are sometimes hitting this in the tests? just a theory right now.

**Comment by rallytime** [comment]
Interesting...what a weird requirement.

**Comment by Ch3LL** [comment]
yeah just a theory at this time. i'll check back to see that the tests are actually getting deleted.

**Comment by Ch3LL** [comment]
okay tests still failed after previous run. will dive into this later today
